kratos/tests/geometries/test_geometry.h

```diff
@@ -28,14 +28,15 @@
 namespace Kratos {
 namespace Testing {
 
+    static std::ofstream out("/dev/null");
```

Comment:
Seems ok, but in long term I would like to know why the same mechanism is not working in the [tester](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/testing/tester.cpp#L259)